src/Language/Marlowe/ACTUS/ActusContracts.hs

```diff
@@ -16,12 +16,12 @@ import Data.Time.Clock.System
 import Debug.Trace
 
 import Language.Marlowe.ACTUS
-import Language.Marlowe
+import Language.Marlowe as M
 
-dayToSlot :: Day -> Slot
-dayToSlot d = let
+dayToTime :: Day -> M.POSIXTime
+dayToTime d = let
     (MkSystemTime secs _) = utcToSystemTime (UTCTime d 0)
-    in Slot (fromIntegral secs - cardanoEpochStart `mod` 20)
+    in POSIXTime (fromIntegral secs - cardanoEpochStart `mod` 20)
```

Comment:
Do you need to reference the `M` here? like in line 21?

```suggestion
    in M.POSIXTime (fromIntegral secs - cardanoEpochStart `mod` 20)
```

Reply:
I think we don't need it because it is the only constructor with that name, otherwise the compiler would have complained. But that works cause I didn't use the `qualified` keyword